static/assets/js/eclipsefdn.adopters.js

```diff
@@ -32,7 +32,8 @@
   var default_options = {
     project_id: "",
     selector: ".eclipsefdn-adopters",
-    ul_classes: ""
+    ul_classes: "",
+    logo_version: "colored"
```

Comment:
I think this would be simplier if it was a bool. Foe example, black_white_logo: true